pkg/minikube/assets/addons.go

```diff
@@ -407,7 +407,7 @@ var Addons = map[string]*Addon{
 		MustBinAsset(
 			"deploy/addons/ambassador/ambassadorinstallation.yaml",
 			vmpath.GuestAddonsDir,
-			"ambassadorinstallation.yaml.yaml",
+			"ambassadorinstallation.yaml",
```

Comment:
was this a bug that this is trying to fix ? if yes please change the PR title

Reply:
updated